README.md

```diff
@@ -172,4 +172,9 @@ I will also accept helpful pull requests if you find an issue or have an idea fo
 
 Last but not least, check out [yWorks](https://www.yworks.com)' fine diagramming SDKs for software developers [yFiles](https://yworks.com/yfiles) and our [free online graph and diagram editors](https://yworks.com/editors)!
 
+
+## Add bot to a channel 
```

Comment:
That doesn't seem to be a good place in the document, IMHO. It should be higher up near installation.